test/integration/master/kms_plugin_mock.go

```diff
@@ -22,9 +22,8 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
-	"net"
-
 	"google.golang.org/grpc"
```

Comment:
Move this import down to the rest of non-stdlib imports

Reply:
Done.